src/components/calcite-chip/calcite-chip.scss

```diff
@@ -62,6 +62,11 @@
   --calcite-chip-button-border-radius: 0 50px 50px 0;
 }
 
+.container {
+  @apply inline-flex
+    items-center;
```

Comment:
It looks like if we add `h-full` here, the close button when in focus will extend the full height of the chip. What do you think?
```css
@apply inline-flex items-center h-full;
```

Reply:
Sounds good to me. 👍  You guys are the design consistency experts :)